app/models/reservation.rb

```diff
@@ -34,7 +34,8 @@ class Reservation < ActiveRecord::Base
   # query by where('flags & ? > 0', FLAGS[:flag])
   # or where('flags & ? = 0', FLAGS[:flag]) for not flagged
   FLAGS = { request: (1 << 1), broken: (1 << 2), lost: (1 << 3),
-            fined: (1 << 4), missed_email_sent: (1 << 5) }
+            fined: (1 << 4), missed_email_sent: (1 << 5),
+            expired: (1 << 6) }
```

Comment:
We also need to update the [wiki page](https://github.com/YaleSTC/reservations/wiki/Reservation-Flags) to reflect the addition of a new flag.